lib/spack/spack/build_environment.py

```diff
@@ -242,9 +240,10 @@ def set_module_variables_for_package(pkg, m):
 def get_rpaths(pkg):
     """Get a list of all the rpaths for a package."""
     rpaths = [pkg.prefix.lib, pkg.prefix.lib64]
-    rpaths.extend(d.prefix.lib for d in pkg.spec.dependencies.values()
+    deps = pkg.spec.dependencies(deptype=spack.DEP_INSTALL)
```

Comment:
this makes sense to me but per the comment above it won't match what is in the `cc` wrapper